api/config/config_local.json

```diff
@@ -5,6 +5,10 @@
     "clusterIdleMaxAgeDays": 3,
     "clusterDefaultMachineType": "n1-standard-4",
     "clusterDefaultDiskSizeGb": 50,
+    "leoRuntimeMaxAgeDays": 7,
```

Comment:
I prefer "notebookRuntime..." for these, considering the name "leo" as an implementation detail

Reply:
done